### ISSUE TITLE
v.2.6.2 (TL-2019 pretest) PostScript error: undefined in findfont

### ISSUE THREAD

**Issue by agrahn** [mgieseki/dvisvgm#102]
The following MWE produces the Ghostscript error:
```
  PostScript error: undefined in findfont
  Operand stack:
      Times-Roman
````

```latex
\documentclass{article}
\pagestyle{empty}

\begin{document}
\makebox[1in][l]{\special{"
 /Times-Roman findfont 10 scalefont setfont 0 0 moveto (Hello!) show
}}
\end{document}
```
Compiled with:
```
latex
latex
dvisvgm --font-format=woff --exact --zoom=-1
```

Some PSTricks packages, such as `pst-solides3d`, make direct use of installed PostScript fonts. For such DVI files no SVG output is produced using the current `dvisvgm` version in TL-2019. It used to work with `dvisvgm-2.3.5` contained in TL-2018.

**Comment by mgieseki** [comment]
This is a [Ghostscript bug](https://bugs.ghostscript.com/show_bug.cgi?id=700290) introduced with version 9.26. There's already a patch available that fixes the issue. Unfortunately, I can't do much from my side.

**Comment by agrahn** [comment]
Ok. But, as far as `dvisvgm` contained in TL is dynamically linked against libgs, both versions of `dvisvgm` that I have should use libgs of my GS-9.26 installation (Gentoo-Linux x86_64).

**Comment by mgieseki** [comment]
In order to support PDF files, I have to call Ghostscript with option `-dDELAYBIND`. Since this feature was only added after the release of TL 2018, the GS bug isn't triggered with older versions of dvisvgm.

**Comment by agrahn** [comment]
Thus, as soon as GS is updated to 9.27 on my system the error will hopefully go away? By the way, using `ldd` command on Linux doesn't list any of `/usr/lib64/libgs.so`, `/usr/lib64/libgs.so.9`, `/usr/lib64/libgs.so.9.26` amongst the dynamically linked libraries. Is it possible that `dvisvgm` in TL is statically linked against libgs? This would be bad, as the error would persist.

**Comment by mgieseki** [comment]
> Thus, as soon as GS is updated to 9.27 on my system the error will hopefully go away? 

Yes, I hope so.

> Is it possible that `dvisvgm` in TL is statically linked against libgs? This would be bad, as the error would persist.

I don't really know how the TL maintainers handle GS dependencies. It's quite possible that some builds provide statically linked binaries. In this case, they should preferably link against a patched GS 9.26 because the bug might affect other utilities coming with TL as well.

**Comment by agrahn** [comment]
Thank you very much. So, someone should inform the TL maintainers about this issue.

**Comment by mgieseki** [comment]
@kberry, @mojca, could you give us some information on how Ghostscript is handled in TL builds? Is it always linked statically? If so, could you apply the GS patch mentioned above to avoid the problem reported by Alexander?

**Comment by kberry** [comment]
libgs is always dlloaded at runtime (not dynamically linked at compile time) in TL-provided binaries. Otherwise the binary would fail to work on some users' machines. You may recall the endless agony we went through on this topic a couple years ago.

**Comment by mgieseki** [comment]
Thanks for refreshing my memory, Karl. Now that you mention it, there have indeed been some problems in this department... 
So, for now the `findfont` bug must be fixed by patching or downgrading Ghostscript on the user's machine. A future GS version 9.27+ should fix it too.

@agrahn, you can apply the [patch](http://git.ghostscript.com/?p=ghostpdl.git;a=commitdiff;h=2756f0efae1d) directly to the corresponding PostScript file which is located in `/usr/share/ghostscript/Resource/Init` on my machine. There's no need to recompile GS.

**Comment by mgieseki** [comment]
The issue seems to be fixed in GS >= 9.27, so I close the ticket here. Feel free to reopen it if necessary.
